client/src/app/employee/employee.component.ts

```diff
@@ -1,20 +1,41 @@
 import { Component } from '@angular/core';
-
-import { faLock, faUnlock } from '@fortawesome/free-solid-svg-icons';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import { Observable } from 'rxjs';
+import { Employee } from './employee-detail/state/employee.model';
+import { EmployeeQuery } from './employee-detail/state/employee.query';
+import { EmployeeService } from './employee-detail/state/employee.service';
+import { EmployeeListItem } from './employee-list/state/employee-list.model';
+import { EmployeeListQuery } from './employee-list/state/employee-list.query';
+import { EmployeeListService } from './employee-list/state/employee-list.service';
 
 @Component({
   selector: 'app-employee',
   templateUrl: './employee.component.html',
   styleUrls: ['./employee.component.scss'],
 })
 export class EmployeeComponent {
-  readonly faLock = faLock;
-  readonly faUnlock = faUnlock;
-  isReadonly = true;
+  readonly faPlus = faPlus;
+  employees: Observable<EmployeeListItem[]>;
+  employee: Observable<Employee>;
+  uiState = '';
+
+  constructor(
+    private employeeQuery: EmployeeQuery,
+    private employeeService: EmployeeService,
+    private employeeListService: EmployeeListService,
+    private employeeListQuery: EmployeeListQuery,
+  ) {
+    this.employee = this.employeeQuery.select((state) => state as Employee);
```

Comment:
I think you can remove this cast `as Employee`.

Also, if you want to reduce the syntax noise you can move the query assignments up to where the fields are declared:

```typescript
  employees = this.employeeListQuery.selectAll();
  employee = this.employeeQuery.select();
```

Reply:
fixed